lib/library.rb

```diff
@@ -0,0 +1,49 @@
+require './lib/author'
+
+class Library
+  attr_reader :name, :authors
+
+  def initialize(name)
+    @name = name
+    @authors = []
+  end
+
+  def add_author(author)
+    @authors << author
+  end
+
+  def books
+    if @authors != []
+      @authors.map do |author|
+        author.books
+      end.flatten
+    else
+      []
+    end
+  end
+
+  def find_author(search_author)
```

Comment:
I'm confused on what this method is doing. You're passing it an author object, then searching an array of author objects to get back the same exact author object? Not needed. :)